documentation/scripts/buildSearchIndex.js

```diff
@@ -43,7 +43,8 @@ async function main() {
         if (fileName !== 'README.md') {
             componentName = fileName.replace('.md', '');
         }
-        const body = await fs.readFile(path, { encoding: 'utf8' });
+        const content = await fs.readFile(path, { encoding: 'utf8' });
+        const body = content.replace(/```((.|\s)*?)```/g, '');
```

Comment:
@andrewhatran I'm not sure whether it's size of the search index, or the specifics of indexing coded content, but the addition of the new documentation was causing trouble in the search interface, this will extract the code snippets from the documentation pages _before_ parsing them for the search index, and causes the interface to work again 🙈. This is probably a sign we're getting close to the edge of the capabilities of `lunr.js`, but a battle for another day...

This also saves about 50KB on the index as well, so working and lighter, win:win!